metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/GenericEnrichmentBolt.java

```diff
@@ -179,6 +180,7 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
   @SuppressWarnings("unchecked")
   @Override
   public void execute(Tuple tuple) {
+    long texecute1 = System.currentTimeMillis();
```

Comment:
I wondered about this also @justinleet and the light searching I did wasn't all that conclusive. Do you by chance have any references?